src/style.css

```diff
@@ -0,0 +1,139 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+/*region Normalize*/
+h2,
+p {
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  background-color: #ccc;
+}
+/*endregion*/
+
+/*region Card*/
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 0 16px;
+
+  --text-title-color: #060b35;
+  --text-text-color: #616070;
+}
+
+.card__picture {
+  width: 160px;
+  height: 134px;
+  margin-top: 32px;
+}
+
+.card__title {
+  padding-top: 40px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--text-title-color);
+}
+
+.card__code {
+  padding-top: 4px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--text-text-color);
+  align-self: flex-start;
+}
+
+.star {
+  padding-top: 16px;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__price {
+  padding-top: 24px;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.card__text {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--text-text-color);
+}
+
+.card__cost {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--text-title-color);
+}
+
+.card__button {
+  margin: 16px 0 17px;
+  padding: 11px 0;
+  width: 100%;
+  background: #00acdc;
+  border: 1px transparent;
+  border-radius: 5px;
+  color: #fff;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-align: center;
+  text-transform: uppercase;
+  text-decoration: none;
+}
+
+.card__button:hover {
+  background: #fff;
+  color: #00acdc;
+  border: 1px solid #00acdc;
+}
+/*endregion*/
+
+/*region Stars*/
+.stars {
+  display: flex;
+}
+
+.stars_review {
+  align-self: flex-end;
+  color: var(--text-title-color);
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background: no-repeat center url("../src/images/star.svg");
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
```

Comment:
you should provide functionality for all possible count of stars (1,2,3,4,5)

Reply:
+